### ISSUE TITLE
Tabs2: Using expander and resizing window causes indicator to hang behind

### ISSUE THREAD

**Issue by samdenicola** [palantir/blueprint#1035]
Not sure how much responsiveness you all are designing for but noticed this bug today. Tab indicator seems to be fine when not using the expander.

### Bug report

- __Package version(s)__: Blueprint Core 1.15
- __Browser and OS versions__: Chrome 58

#### Steps to reproduce

1. Add a Tabs2.Expander to push tabs to the right of your Tabs2 component
2. Open browser window at half size
3. Resize browser window to full screen
```
<Tabs2
          id="PlateAndTableView"
          onChange={this.handleTabChange}
          renderActiveTabPanelOnly={true}
        >
          <h2> Plate Contents </h2>
          <Tabs2.Expander />
          <Tab2 id="plate" title="Plate View" panel={<PlateWellsViewContainer />} />
          <Tab2 id="table" title="Table View" panel={<TablePanel />} />
</Tabs2>
```
#### Actual behavior

<img width="1136" alt="screen shot 2017-04-25 at 11 09 00 am" src="https://cloud.githubusercontent.com/assets/13247167/25400529/c325b4a8-29a7-11e7-9f21-2aaf420b3b7a.png">

#### Expected behavior

<img width="1133" alt="screen shot 2017-04-25 at 11 10 34 am" src="https://cloud.githubusercontent.com/assets/13247167/25400554/d5995720-29a7-11e7-9205-0b580ad51fc2.png">

**Comment by giladgray** [comment]
yep definitely a thing... question is if i should _throttle_ or _debounce_ the resize event. throttling causes the tab indicator to lag behind the tab by a few frames; debouncing would only update the indicator _after_ resizing is completed. my preference is debounce because it'll only update once at the end, rather than offering several subpar intermediate states.

#### throttle
![tabs2-resize-throttle](https://cloud.githubusercontent.com/assets/464822/25831453/110d4fca-3419-11e7-89c6-4bf02082fbe0.gif)

#### debounce (100ms)
![tabs2-resize-debounce](https://cloud.githubusercontent.com/assets/464822/25831457/18caac76-3419-11e7-846c-0572403e1e0c.gif)

**Comment by samdenicola** [comment]
@giladgray 
Makes sense to go with debounce for performance -- although I think the throttle is visually more appealing.

What happens when you use a hotkey to resize the window to half-size with debounce? That is the most common resize I use.

**Comment by llorca** [comment]
🤣 That's fun!

FWIW, I also prefer the visuals of the throttle

**Comment by giladgray** [comment]
ooh i'll try hotkeys, but i'm pretty sure it'll work flawlessly and move into place after 100ms
